### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by amyblais**
#### Summary
Updating contributor process - work in progress.

### CODE REVIEW

**Comment by amyblais** [comment]
@jasonblais 

 - Corey had two question marks in his draft and I left them since I wasn't sure if they will be discussed.
 - I incorporated Corey's draft, but let me know if I was meant to replace the original text and I can remove that text.
 - Who will be assigned to review this PR?
 - Here are additional comments that Corey had in his draft:
   - "Describe the various other labels somewhere" (AB: are there others than the ones in this PR?)
   - "Unify all the repos to have the same labels (??)"
   - "George had mentioned an idea proposed by Jono which is once PRs reach a certain stage they get assigned shepherds who's responsibilities include being the person who helps the submitter and sometimes finishes off critical PRs. I like this idea and George is going to ping Jono for more info."

**Comment by jasonblais** [comment]
@amyblais 

> Corey had two question marks in his draft and I left them since I wasn't sure if they will be discussed.

For PM assignment, we can just keep it simple for now and say `Assign PRs related to end user features to @esethna, others to @jasonblais`. It's roughly our current breakdown, and we can adjust it if needed

For line 51, I think that sounds fine.

> I incorporated Corey's draft, but let me know if I was meant to replace the original text and I can remove that text.

No, this is perfect. The only modification I would make is to keep the stages the same, i.e. have PM review as stage 1, dev review as stage 2, etc. This way the stages match the GitHub labels.

> Who will be assigned to review this PR?

I can be the reviewer.

> Here are additional comments that Corey had in his draft:

For describing the othes labels, we have some at the top of the doc, e.g. what `Major Change` label is. So what we have now is fine.

For the other points, perhaps you can make notes of them and consider if these are something we want to do in coming weeks as you start working on the contributor and help wanted process?

**Comment by amyblais** [comment]
@jasonblais Made the edits and this is ready for review/merge.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,51 +13,63 @@ The one exception may be around release time, where the review process may take
...
- - Set for next release if the PM thinks there is enough time for the PR to be merged and sufficiently tested on `master` before code complete.
- - Set for a future release if PR is too large to test prior to the code complete date
-   - PM responds to submitter letting them know that PR may have a delay in review due to the release cycle
-
-Next, the PM tests changes on the spinmint:
+Next, the PM tests changes on a spinmint test server:
+ - PM tests and verifies pull requests via the `Setup Test Server` label. Initial review is completed within 2 business days.
  - If changes are required, PM submits review as "Changes Requested", with a comment on the areas that require updates. Comment explains why changes are needed linking back to design principles.
-   - PM applies `Awaiting Submitter Action` label to more easily query the PR queue
+   - PM applies `Awaiting Submitter Action` label to more easily query the PR queue.
```
Comment (jwilander) [inline CONTRIBUTING.md]: Still don't like this and think it's unnecessary

**Comment by amyblais** [comment]
I don't know how to rebase a PR, so I'll open a new one and hope that one works.

**Pull request closed without merging**